indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -127,6 +127,50 @@ function EditableListDisplay({
     [editors]
   );
 
+  const filterOptions = useMemo(
+    () => [
+      {
+        key: 'state',
+        text: Translate.string('Status'),
+        options: [
+          {value: 'accepted', text: Translate.string('Accepted')},
+          {value: 'ready_for_review', text: Translate.string('Ready for review')},
+          {value: 'not_submitted', text: Translate.string('Not submitted')},
+        ],
+      },
+      {
+        key: 'editor',
+        text: Translate.string('Editor'),
+        options: _.uniqBy(
+          contribsWithEditables.map(x => x.editable.editor).filter(Boolean),
+          'id'
+        ).map(({identifier, fullName}) => ({
+          value: identifier,
+          text: fullName,
+        })),
+      },
+      {
+        key: 'code',
+        text: Translate.string('Program code'),
+        options: _.uniq(contribList.map(x => x.code).filter(Boolean)).map(code => ({
+          value: code,
+          text: code,
+        })),
+      },
+      {
+        key: 'keywords',
+        text: Translate.string('Keywords'),
+        options: _.uniq(contribList.reduce((acc, curr) => [...acc, ...curr.keywords], [])).map(
```

Comment:
Wouldn't this do the same as the reduce? `_.flatten(contribList.map(c => c.keywords));`